activemodel/lib/active_model/errors.rb

```diff
@@ -496,17 +509,8 @@ def generate_message(attribute, type = :invalid, options = {})
 
       key = defaults.shift
       defaults = options.delete(:message) if options[:message]
-      value = (attribute != :base ? @base.send(:read_attribute_for_validation, attribute) : nil)
-
-      options = {
-        default: defaults,
-        model: @base.model_name.human,
-        attribute: @base.class.human_attribute_name(attribute),
-        value: value,
-        object: @base
-      }.merge!(options)
 
-      I18n.translate(key, options)
+      I18n.translate(key, { default: defaults }.merge!(options))
```

Comment:
Weren't these options already merged in on 489?

Reply:
This merge is just to add the `default` key.

Reply:
If it is only to add the default keys I think we should add it using `[]=`

Reply:
I'll change it